sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsOperationStatus.cs

```diff
@@ -12,5 +11,10 @@ namespace Azure.AI.TextAnalytics
     [CodeGenModel("State")]
     public partial struct TextAnalyticsOperationStatus
     {
+        [CodeGenMember("PartiallyCompletedValue")]
+        private const string PartiallySucceededValue = "partiallySucceeded";
+
+        /// <summary> partiallyCompleted. </summary>
+        public static TextAnalyticsOperationStatus PartiallySucceeded { get; } = new TextAnalyticsOperationStatus(PartiallySucceededValue);
```

Comment:
curious, is this one not present in the swagger?

Reply:
Yeah, this is not in swagger. Now i am curious how are other handling this edge case

@abhahn, @deyaaeldeen @mssfang

Reply:
Could u open an issue on the service side so it gets added?

Reply:
created - https://msazure.visualstudio.com/Cognitive%20Services/_workitems/edit/9296630